src/Filesystem/Filesystem.php

```diff
@@ -102,4 +102,22 @@ public function remove($files)
             }
         }
     }
+
+    /**
+     * Returns whether the file path is an absolute path.
+     *
+     * @param string $file A file path
+     *
+     * @return bool
+     */
+    public function isAbsolutePath(string $file): bool
```

Comment:
is this a time to add a new dependency? :D

Reply:
lol, expected question. I still don't know.

Reply:
I think it's time to, it's the 3rd method we are copy-pasting from it :) Besides it's not like filesystem is a huge dependency, it's quite slim and don't change much

Reply:
@borNfreee let's do this. If you don't want to do it in the current PR let's do it as separate.

Reply:
got it, I will do it in this PR